### ISSUE TITLE
Feature request: Keyboard shortcut for "I am feeling lucky" radio

### ISSUE THREAD

**Issue by PunchyRascal** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#2676]
Hey there!

It would be great if it would be possible to set a keyboard shortcut to start/reload the "I am feeling lucky" radio. For me that's most of what I listen to, but sometimes it plays some crap and I have to reload it several times after listening for a few seconds, so putting some of those "multimedia" keyboard buttons to use would be very helpful. Thanks :-)

**Comment by jostrander** [comment]
I was doing some quick testing with this in the dev tools console, and I might have found a possible solution to making this happen. Normally, this wouldn't be feasible because the IFL button is only on the home page of GPM. However, if we clone the DOM element and apply it off screen and at the end of the DOM it will retain its polymer event handlers and work when pseudo clicked as in with a hotkey. The only hang up is,  you would have to always visit the home page when the application started so it could grab the original IFL button and clone it. @MarshallOfSound thoughts?

**Comment by jostrander** [comment]
Actually thanks to polymer being polymer. We can insert a 'gpm-ifl-button' node into the dom and hide it and it will work on any page, not needing the home page to be ever seen.

**Comment by jostrander** [comment]
Proof of concept:

```
const button = document.createElement('gpm-ifl-button');
button.style.display = 'none';
document.body.appendChild(button);
```
then when you want to trigger IFL, 
```
button.click();
```

**Comment by PunchyRascal** [comment]
Awesome!

**Comment by jostrander** [comment]
Upstream PR gmusic-utils/gmusic.js#21

**Comment by chuckfecht** [comment]
This was implemented, the issue should be closed.

**Comment by MarshallOfSound** [comment]
> This was implemented, the issue should be closed.

I don't remember this being implemented, can you link the PR? 🤔

**Comment by Arlodotexe** [comment]
My bad, I hadn't looked closely enough at PR#21

**Comment by matijaerceg** [comment]
Not sure if I should start a new issue, but if non-homepage 'feeling lucky radio' is now possible, it would be nice for it to be added to the system tray right click menu as well.

**Comment by jostrander** [comment]
This has been implemented in #2914
